File: src/Java/autor/autor.java
package Java.autor;

public class autor {
    String nombre;
    String nacionalidad;

    public autor(String nombre, String nacionalidad) {
        this.nombre = nombre;
        this.nacionalidad = nacionalidad;
    }

    @Override
    public String toString() {
        return "Nombre: " + this.nombre + "\nNacionalidad: " + this.nacionalidad;
    }
}

File: src/C#/Parcial1/autor/autor.cs
using System;

public class Autor {
    public string Name { get; set; }
    public string Nacionalidad { get; set; }

    public Autor(string nombre, string nacionalidad) {
        this.Name = nombre;
        this.Nacionalidad = nacionalidad;
    }

    public override string ToString() {
        return "Nombre: " + this.Name + "\nNacionalidad: " + this.Nacionalidad;
    }
}
